01/app.js

```diff
@@ -0,0 +1,23 @@
+const form = document.querySelector('form');
+const submit = document.querySelector('input[type=submit]');
+
+const getValue = function(e) {
+    e.preventDefault();
+    let liContent = '';
+    const ul = document.querySelector('.users-list');
+    for(const el of form.elements){
+      if(el.getAttribute('type') != 'submit'){
+        liContent = `${liContent} ${el.value}`
+        el.value = "";
+      }   
+    }
```

Comment:
Fajne rozwiązanie, ale raczej mało praktyczne i mało czytelne.
Czy jeśli pojawi się kolejny element formularza to on zostanie po prostu dodany do niego czy pewnie będzie inaczej osadzony? Pewnie to drugie więc i tak ten element będzie trzeba zmodyfikować. Dlatego lepiej napisać jawnie jakie elementy pobieramy :)